rdrf/rdrf/management/commands/create_translation_file.py

```diff
@@ -0,0 +1,305 @@
+import yaml
+import sys
+import os.path
+import re
+from rdrf.utils import de_camelcase
+from rdrf.models import Registry
+from tempfile import TemporaryDirectory
+from django.core.management import BaseCommand
+from django.core.management import call_command
+
+class Command(BaseCommand):
```

Comment:
For the class:
  * it isn't very clear to me how to use the command. Maybe a bit more of high level documentation or usage notes?
  * IMO, too much is handled in this class